src/media/css/newsletter.styl

```diff
@@ -134,6 +134,16 @@
     .newsletter-select {
         width: 50%;
     }
+    .newsletter-email-field {
+        +rtl() {
+            margin: 0;
```

Comment:
Looks like this is originally set on L88, can it move there and be `margin: 0 0 0 5px;` instead? I added some `+ltr()` mixins on another branch that might help here also. So you could do this:

```styl
.newsletter-email-field {
  +ltr() {
    margin-right: 5px;
  }
  +rtl() {
    margin-left: 5px;
  }
}